File: app/src/main/java/com/example/advancedprayertimes/Logic/Entities/API_Entities/Diyanet/DiyanetUlkeEntity.java
package com.example.advancedprayertimes.Logic.Entities.API_Entities.Diyanet;

import com.google.gson.annotations.SerializedName;

public class DiyanetUlkeEntity
{
    // region fields

    @SerializedName("UlkeID")
    private String _ulkeID = null;

    @SerializedName("UlkeAdiEn")
    private String _ulkeAdiEn = null;

    // endregion fields

    // region getter & setter

    public String getUlkeID()
    {
        return _ulkeID;
    }

    public void setUlkeID(String ulkeID)
    {
        _ulkeID = ulkeID;
    }

    public String getUlkeAdiEn()
    {
        return _ulkeAdiEn;
    }

    public void setUlkeAdiEn(String ulkeAdiEn)
    {
        _ulkeAdiEn = ulkeAdiEn;
    }

    // endregion getter & setter
}

File: app/src/main/kotlin/com/example/advancedprayertimes/logic/api_entities/diyanet/DiyanetUlkeEntity.kt
package com.example.advancedprayertimes.logic.api_entities.diyanet

import com.google.gson.annotations.SerializedName

class DiyanetUlkeEntity : AbstractDiyanetSubEntity() {

    @SerializedName("UlkeID")
    var ulkeID: String? = null

    @SerializedName("UlkeAdiEn")
    var ulkeNameEn: String? = null

    override val id: String?
        get() = ulkeID

    override val nameEn: String?
        get() = ulkeNameEn
}